adapter/internal/adapter/adapter.go

```diff
@@ -196,6 +197,13 @@ func Run(conf *config.Config) {
 		})
 	}
 
+	// Start the metrics server
+	if conf.Adapter.Metrics.Enabled && conf.Adapter.Metrics.Type == metrics.PrometheusMetricType {
```

Comment:
Isn't it better to consider ignoring the case here ?
`prometheus` , `Prometheus` , `PROMETHEUS`

Reply:
Fix [de37947](https://github.com/wso2/product-microgateway/pull/3428/commits/de379477a8a20379d29d4fa4a4e0bc48d3e0fb18)